getting-started/README.md

```diff
@@ -4,22 +4,143 @@ Amber is designed to make common web-development tasks simple, and easy, maintai
 
 The goal of this overview is to introduce the general concepts in Amber and give you a quick overview of how those concepts are implemented in Amber.
 
-## Conventions Over Configuration
+## Guide Assumptions
 
-Amber provides a basic organizational structure that covers class names, filenames, database table names, and other conventions. While the conventions take some time to learn, by following the conventions Amber provides you can avoid needless configuration and make a uniform application structure that makes working with various projects simple.
+This guide is designed for beginners who want to get started with a Amber application from scratch. It does not assume that you have any prior experience with Amber. However, to get the most out of it, you need to have some prerequisites installed:
 
-## Model-View-Controller 
+* The [Crystal ](https://crystal-lang.org/)language version 0.23.1 or higher
+* The [Shards](https://github.com/crystal-lang/shards) packaging system, which is installed with Crystal by default. 
+* A working installation of the [SQLite3 Database](https://www.sqlite.org/)
 
-Model–view–controller (MVC) is a software architectural pattern for implementing user interfaces on computers. It divides a given application into three interconnected parts. This is done to separate internal representations of information from the way information is presented to, and accepted from, the user. The MVC design pattern decouples these major components allowing for efficient code reuse and parallel development.
+Amber is a web application framework running on the Crystal programming language. If you have no prior experience with Crystal, you will find a very steep learning curve diving straight into Amber. There are several curated lists of online resources for learning Crystal:
```

Comment:
Amber is a web application framework written in the Crystal programming language. If you have experience with Crystal, it may make using the guide a bit easier.

If you are coming from another language, Crystal is easy to pickup with syntax similar to Ruby. However, it is statically typed and compiled. If you have never worked with that kind of language, spending some time to learn crystal will help:

* [Official Learn Crystal Docs](https://crystal-lang.org/docs/overview/)
* [Crystal Exercisms](http://exercism.io/languages/crystal/about)